core/web/helpers.go

```diff
@@ -42,7 +42,7 @@ func paginatedResponse(
 	err error,
 ) {
 	if err == orm.ErrorNotFound {
```

Comment:
I'd like to see a test for this new behaviour and I'd also like to see it observe the above, `page==0 && records==0`: return empty record, `page>0 && `records<page*records_per_page`, 404.

Reply:
Yeah, the PR was only demonstrational, but you've answered my question above